treeherder/log_parser/management/commands/crossreference_error_lines.py

```diff
@@ -55,12 +57,16 @@ def handle(self, *args, **options):
 
         text_log_errors = TextLogError.objects.filter(
             step__job__guid=job_guid).order_by('line_number')
-        self.crossreference_error_lines(repository,
-                                        job_guid,
-                                        failure_lines,
-                                        text_log_errors,
-                                        bug_suggestions)
-
+        try:
+            self.crossreference_error_lines(repository,
+                                            job_guid,
+                                            failure_lines,
+                                            text_log_errors,
+                                            bug_suggestions)
+        except IntegrityError:
+            logger.error("IntegrityError crossreferencing error lines for job %s" % job_guid)
```

Comment:
Should this be just a warning, given that we could expect this to happen sometimes?